advanced/helm/ei-pattern-1/templates/analytics-worker-deployment-2.yaml

```diff
@@ -163,7 +163,7 @@ spec:
             mountPath: /usr/share/logstash/plugins/
         env:
           - name: NODE_ID
-            value: {{ default "wso2ei-node" .Values.wso2.centralizedLogging.logstash.indexNodeID.wso2EINode }}
+            value: {{ .Release.Name }}-analytics-worker-node1
```

Comment:
@Savidude shall we make this `{{ .Release.Name }}-analytics-worker-node-1` (adds a `-` before the number)? Applies to other instances, as well.

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/200/commits/858b72938c9595c8ccf0c7ba5047b8ff1758ae76